### ISSUE TITLE
Upgrading to Gradle 8.x

### ISSUE THREAD

**Issue by flechajm** [jonasbark/flutter_in_app_update#116]
Hello there, Jonas! 
First of all, excelent package, it's really usefull!

I have a situation here, when I tried to upgrade from Gradle 7.x to 8.x, I got this message:
```
> Configure project :in_app_update
The RepositoryHandler.jcenter() method has been deprecated. This is scheduled to be removed in Gradle 8.0. JFrog announced JCenter's sunset in February 2021. Use mavenCentral() instead.
```

I think you should change this section in your **android/build.gradle**:

```
buildscript {
    ext.kotlin_version = 'x.x.xx'

    repositories {
        google()
        jcenter() <------- HERE
    }
    ...
 }
```

To this:
```
buildscript {
    ext.kotlin_version = 'x.x.xx'

    repositories {
        google()
        mavenCentral() <------- HERE
    }
    ...
 }
```

I hope you can made this change so I can upgrade my project ASAP at my work.
Thanks in advance!

**Comment by xVemu** [comment]
#120

**Comment by jonasbark** [comment]
Released as of 4.2.3
